### PR TITLE
chore: Update Readme

### LINKED ISSUE #814: Broken yarn start &
https://github.com/ubiquity/ubiquity-dollar/blob/17e79d31bca7a9c573990c6fec107f0ec1384ede/README.md?plain=1#L26

Usage Error: Couldn't find a script named "start".

$ yarn run [--inspect] [--inspect-brk] [-T,--top-level] [-B,--binaries-only] <scriptName> ...

### PR DESCRIPTION
**Pull request by molecula451**
Resolves #814

This PR is a short update to the readme, but it updates some things

1. It updates the commands sections and what its need it to execute the repo successfully
2. It deprecate unnecessary private keys
3. It adds a few verbosity on some of the section
4. It also updates a bit the wording

### CODE REVIEW

**Comment by gitcoindev** [comment]
Hi @molecula451 is this still a draft pull request or can be reviewed?

**Comment by molecula451** [comment]
draft but you are welcome to leave suggestions

File: README.md [hunk 1]
```diff
@@ -21,11 +21,12 @@ Introducing the flagship product of [Ubiquity DAO](https://ubq.fi/). The Ubiquit
 
 git clone https://github.com/ubiquity/ubiquity-dollar.git
 cd ubiquity-dollar/
-yarn
+yarn # fetch dependencies
 yarn build:all # builds the smart contracts and user interface
-yarn start & # starts the user interface and daemonize'd to continue to run tests in the background
+yarn start # starts the user interface and daemonize'd to continue to run tests in the background
```
Comment (gitcoindev) [inline README.md:33]: Currently this command fails on a clean checkout as the "start" script definition is not available in the top level package.json and is defined in packages/dapp only.

Comment (gitcoindev) [reply README.md:33]: Additionally, when we remove `&` the dapp starts in the foreground mode, therefore everything written after "starts the user interface" can be removed. We should explicitly add though that this command runs in foreground and another shell / console must be open for subsequent commands to run in parallel to the user interface.

**Review by gitcoindev — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (molecula451) [reply README.md:33]: as this is a bound to change, yarn start it's available under /dapp, which make it practical to use

**Review by rndquu — APPROVED** [review]

**Review by gitcoindev — APPROVED** [review]

**Pull request merged**